### PR TITLE
gdal 3.7.0

### PR DESCRIPTION
**Pull request by e-n-f**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/gdal.rb [hunk 1]
```diff
@@ -59,6 +58,7 @@ class Gdal < Formula
   depends_on "python@3.11"
   depends_on "qhull"
   depends_on "sqlite"
+  depends_on "swig"
```
Comment (carlocab) [inline Formula/gdal.rb:61]: This is probably a `:build` dependency.

Please also squash your commits together so you only have one commit.

Thanks!

Comment (e-n-f) [reply Formula/gdal.rb:61]: Thanks, I'll fix that.

File: Formula/gdal.rb [hunk 2]
```diff
@@ -59,6 +59,7 @@ class Gdal < Formula
   depends_on "python@3.11"
   depends_on "qhull"
   depends_on "sqlite"
+  depends_on "swig" => :build
```
Comment (carlocab) [inline Formula/gdal.rb:62]: Needs to be listed with the build dependencies above. (Squash commits again please, thanks.)

Comment (e-n-f) [reply Formula/gdal.rb]: Thanks. Fixed and squashed.

**Comment by carlocab** [comment]
Applies fixes for pdal and postgis. Not sure how to deal with the others. osmcoastline looks like it needs a patch like pdal's.

Old logs: https://github.com/Homebrew/homebrew-core/actions/runs/5096532937

File: Formula/postgis.rb [hunk 3]
```diff
@@ -48,6 +48,7 @@ def postgresql
 
   def install
     ENV.deparallelize
+    ENV.append "CXXFLAGS", "-std=c++14" # Needed for GDAL 3.7+
```
Comment (carlocab) [inline Formula/postgis.rb:51]: We need C++17, it seems.

**Comment by e-n-f** [comment]
Thanks for the fixes! I'll see if i can figure out how to patch `osmcoastline`.

File: Formula/postgis.rb (hunk 3 above)
Comment (e-n-f) [reply Formula/postgis.rb:51]: Addressed in ca731c61009511f35e98b7def8db09dff43a4ac2

**Comment by carlocab** [comment]
Any luck with `osmcoastline`? At worse we can probably report the problem to them, even if we don't have a patch yet.

**Comment by e-n-f** [comment]
Looks like `osmcoastline` has been fixed upstream in https://github.com/osmcode/osmcoastline/commit/10e2eaa8af376bc4de08c1500d444f9a58e75f67.

I just filed https://github.com/DroneDB/DroneDB/pull/374 to fix DroneDB.

Looking at `simple-tiles` now.

**Comment by e-n-f** [comment]
Actually I forgot I had already added the patch for `simple-tiles`. I'll add the patches for `osmcoastline` and `dronedb` and then I think this should work.

**Comment by e-n-f** [comment]
@carlocab Is it possible to reopen this PR? Or should I make a new one?

The last commit I pushed (3207d9f4ef8576a1) should fix `osmcoastline` and `dronedb`.

I am confused about `simple-tiles` because it builds for me with the `master` branch, or by applying the patches named in `simple-tiles.rb` with `git apply`, but `brew install --force --build-from-source --debug Formula/simple-tiles.rb` fails. The binary data in the `waf` file is corrupted in the patching and gives an error (`Error: corrupt archive (2)`). I will keep trying to work around it, but it looks like the error was already present in the previous `simple-tiles.rb`, not through this PR, so I don't understand how it ever worked.

**Comment by chenrui333** [comment]
#134485

**Pull request closed without merging**